controllers/provisioning_controller.go

```diff
@@ -270,6 +271,15 @@ func (r *ProvisioningReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 
 // SetupWithManager configures the manager to run the controller
 func (r *ProvisioningReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	namespacedName := types.NamespacedName{Name: BaremetalProvisioningCR}
+	baremetalConfig, err := r.readProvisioningCR(namespacedName)
+	if err != nil || baremetalConfig == nil {
+		err = r.updateCOStatus(ReasonComplete, "nothing to do in assisted install", "")
```

Comment:
I think we want to not mention assisted. Conceivably, someone could install a compact 3-node baremetal cluster without provisioning services by deleting the CR.

So maybe something like "Provisioning CR not found on baremetal platform; marking operator as available"?

Reply:
I did not like that string. Thanks for the suggestion.